### DISCUSSION TITLE
Documentation, update request

### DISCUSSION THREAD

**Discussion by 9to1url** [narrowlink/narrowlink#68] [General]
I found this in document , link: https://narrowlink.com/docs/comparisons/tailscale

```
For example, the Tailscale client binary size is 46MB (tailscale 18MB, tailscale 28MB)
```
Maybe one of the tailscale 28MB you mean tailscale-server? :-)

**Comment by 9to1url** [comment]
LICENSE file in gateway folder is MPL but should be AGPL, could you update that file? thanks

**Comment by SajjadPourali** [comment]
Thank you, I just updated them.
